### PR TITLE
feat: Optimize middlewares

### PR DESCRIPTION
**Pull request by hwbrzzl**
Closes # 

## 📑 Description

Add tls, cors middleware.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: cors_test.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	configmocks "github.com/goravel/framework/contracts/config/mocks"
+	contractshttp "github.com/goravel/framework/contracts/http"
+	"github.com/stretchr/testify/assert"
+)
```
Comment (devhaozi) [inline cors_test.go:11]: Please sort import.

File: cors.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package gin
+
+import (
+	nethttp "net/http"
+
+	httpcontract "github.com/goravel/framework/contracts/http"
+	"github.com/rs/cors"
+)
+
```
Comment (devhaozi) [inline cors.go:9]: Please sort import.

File: tls_test.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+import (
+	"crypto/tls"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	configmocks "github.com/goravel/framework/contracts/config/mocks"
+	contractshttp "github.com/goravel/framework/contracts/http"
+	"github.com/stretchr/testify/assert"
```
Comment (devhaozi) [inline tls_test.go:11]: Please sort import.

File: tls.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
+package gin
+
+import (
+	"net/http"
+
+	contractshttp "github.com/goravel/framework/contracts/http"
+	"github.com/unrolled/secure"
```
Comment (devhaozi) [inline tls.go:7]: Please sort import.

File: group_test.go [hunk 5]
```diff
@@ -5,6 +5,7 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/gin-gonic/gin"
 	configmock "github.com/goravel/framework/contracts/config/mocks"
 	httpcontract "github.com/goravel/framework/contracts/http"
 	"github.com/goravel/framework/contracts/route"
```
Comment (devhaozi) [inline group_test.go:11]: Please sort import.

File: group.go [hunk 6]
```diff
@@ -2,14 +2,16 @@ package gin
 
 import (
 	"net/http"
+	"strings"
 
 	"github.com/gin-gonic/gin"
-
+	"github.com/goravel/framework/contracts/config"
 	httpcontract "github.com/goravel/framework/contracts/http"
 	"github.com/goravel/framework/contracts/route"
```
Comment (devhaozi) [inline group.go:10]: Please sort import.

**Review by devhaozi — CHANGES_REQUESTED** [review]

File: cors_test.go (hunk 1 above)
Comment (hwbrzzl) [reply cors_test.go:11]: `github.com/goravel/framework` and `github.com/stretchr/testify` are both third packages, so they can be together. The import order is: 
```
system packages

third packages

self packages
```

**Review by devhaozi — APPROVED** [review]
LGTM

**Pull request merged**